armi/bookkeeping/db/tests/test_database3.py

```diff
@@ -365,39 +371,6 @@ def test_loadSortSetting(self):
         self.assertEqual(len(r0), len(r1))
         self.assertEqual(len(r0.core), len(r1.core))
 
-    def test_load_updateGlobalAssemNum(self):
-        from armi.reactor import assemblies
-        from armi.reactor.assemblies import resetAssemNumCounter
-
-        self.makeHistory()
-
-        resetAssemNumCounter()
-        self.assertEqual(assemblies._assemNum, 0)
-
-        r = self.db.load(0, 0, allowMissing=True, updateGlobalAssemNum=False)
-        #  len(r.sfp) is zero but these nums are still reserved
-        numSFPBlueprints = 4
-        expectedNum = len(r.core) + numSFPBlueprints
-        self.assertEqual(assemblies._assemNum, expectedNum)
-
-        # now do the same call again and show that the global _assemNum keeps going up.
-        # in db.load, rector objects are built in layout._initComps() so the global assem num
-        # will continue to grow (in this case, double).
-        self.db.load(0, 0, allowMissing=True, updateGlobalAssemNum=False)
-        self.assertEqual(assemblies._assemNum, expectedNum * 2)
-
-        # now load but set updateGlobalAssemNum=True and show that the global assem num
-        # is updated and equal to self.r.p.maxAssemNum + 1 which is equal to the number of
-        # assemblies in blueprints/core.
-        r = self.db.load(0, 0, allowMissing=True, updateGlobalAssemNum=True)
-        expected = len(self.r.core) + len(self.r.blueprints.assemblies.values())
-        self.assertEqual(15, expected)
-
-        # repeat the test above to show that subsequent db loads (with updateGlobalAssemNum=True)
-        # do not continue to increase the global assem num.
-        self.db.load(0, 0, allowMissing=True, updateGlobalAssemNum=True)
-        self.assertEqual(15, expected)
```

Comment:
A lot of this Assembly number management was designed to allow loading from a db multiple times in the same process. Can we maybe rename the test to be "loading multiple times works as expected" to cover the desire even though the implementation is changing? we don't want a regression in this capability.